.github/workflows/conda-package.yml

```diff
@@ -0,0 +1,290 @@
+name: Conda package
+
+on: push
+
+env:
+  PACKAGE_NAME: mkl_umath
+  MODULE_NAME: mkl_umath
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: ['3.10', '3.11', '3.12']
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set pkgs_dirs
+        run: |
+          echo "pkgs_dirs: [~/.conda/pkgs]" >> ~/.condarc
+
+      - name: Cache conda packages
+        uses: actions/cache@v3
+        env:
+          CACHE_NUMBER: 0  # Increase to reset cache
+        with:
+          path: ~/.conda/pkgs
+          key:
+            ${{ runner.os }}-conda-${{ env.CACHE_NUMBER }}-python-${{ matrix.python }}-${{hashFiles('**/meta.yaml') }}
+          restore-keys: |
+            ${{ runner.os }}-conda-${{ env.CACHE_NUMBER }}-python-${{ matrix.python }}-
+            ${{ runner.os }}-conda-${{ env.CACHE_NUMBER }}-
+
+      - name: Add conda to system path
+        run: echo $CONDA/bin >> $GITHUB_PATH
+
+      - name: Install conda-build
+        run: conda install conda-build
+
+      - name: Build conda package
+        run: |
+          CHANNELS="-c conda-forge -c https://software.repos.intel.com/python/conda --override-channels"
+          VERSIONS="--python ${{ matrix.python }}"
+          TEST="--no-test"
+          echo "CONDA_BLD=${CONDA}/conda-bld/linux-64" >> $GITHUB_ENV
+
+          conda build \
+            $TEST \
+            $VERSIONS \
+            $CHANNELS \
+            conda-recipe-cf
+
+      - name: Upload artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{ env.PACKAGE_NAME }} ${{ runner.os }} Python ${{ matrix.python }}
+          path: ${{ env.CONDA_BLD }}/${{ env.PACKAGE_NAME }}-*.tar.bz2
+
+  test:
+    needs: build
+    runs-on:  ${{ matrix.runner }}
+
+    strategy:
+      matrix:
+        python: ['3.10', '3.11', '3.12']
+        experimental: [false]
+        runner: [ubuntu-latest]
+    continue-on-error: ${{ matrix.experimental }}
+    env:
+      CHANNELS: -c conda-forge -c https://software.repos.intel.com/python/conda --override-channels
```

Comment:
Should this be flipped to ensure that if .condarc has strict channel priority set to true, that packages from software.repos.intel.com have higher priority?

Reply:
I actually want `conda-forge` to take precedence in this workflow, to avoid installing `mkl_umath` into the host environment where I am trying to build `mkl_umath`